### ISSUE TITLE
static asset imports are resolved from absolute (user) path in build 

### ISSUE THREAD

**Issue by samuelstroschein** [brillout/vite-plugin-ssr#794]
### Description

[Static asset handling](https://vitejs.dev/guide/assets.html) resolves absolute paths on build, leading to unresolved assets.  

@brillout you must have changed static asset handling between version `0.4.70` and `0.4.114`. See my related commit that fixed the issue https://github.com/inlang/inlang/pull/565/commits/be8cb84a80791b8e4dc48066892b7ef2586004bd.

![CleanShot 2023-04-19 at 23 22 17@2x](https://user-images.githubusercontent.com/35429197/233203156-efa6f759-32a7-4f33-be1d-b47824b1f328.png)

**Version 0.4.70's dist output**

![CleanShot 2023-04-19 at 23 20 56@2x](https://user-images.githubusercontent.com/35429197/233202966-6074b51c-5912-45a9-8e63-9589749b7c75.png)

**Version 0.4.114's dist output**

![CleanShot 2023-04-19 at 23 32 17@2x](https://user-images.githubusercontent.com/35429197/233205007-ae4e5cb0-0238-4b94-a104-6274cb760124.png)

**Reproduction**

_I assume you already know what change caused this at this point. Hence, I did not invest time in a proper reproduction._

1. checkout commit af71a944d717bd5bd53bad1017ad3501babc1627 from https://github.com/inlang/inlang
2. `npm i`
3. `npm run build`
4. the dist output contains the wrong paths

**Comment by brillout** [comment]
Update to the latest version and insert the following between line 40-41 at `node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/distFileNames.js`.
```js
     console.log(dir, name);
```

What do you get?

**Comment by samuelstroschein** [comment]
Inserted `console.log({dir, name})` for better readability

```
vite v3.2.5 building for production...
transforming (181) src/pages/index/repositories.ts{
  dir: 'assets/static',
  name: '/Users/samuel/Documents/GitHub.nosync/inlang/source-code/website/src/pages/index/assets/editor-image.png'
}
{
  dir: 'assets/static',
  name: '/Users/samuel/Documents/GitHub.nosync/inlang/source-code/website/src/pages/index/assets/cli-image.png'
}
transforming (201) ../git-sdk/dist/api/index.js{
  dir: 'assets/static',
  name: '/Users/samuel/Documents/GitHub.nosync/inlang/source-code/website/src/pages/index/assets/extension-video.mp4'
}
transforming (221) src/pages/index/assets/extension-video.mp4{
  dir: 'assets/static',
  name: '/Users/samuel/Documents/GitHub.nosync/inlang/source-code/website/src/pages/index/assets/editor-video.mp4'
}
✓ 2723 modules transformed.
```

<img width="1376" alt="CleanShot 2023-04-20 at 00 39 59@2x" src="https://user-images.githubusercontent.com/35429197/233215046-3ca03e74-2e48-45a4-8464-86d5b5c37e37.png">

**Comment by samuelstroschein** [comment]
@brillout are you sure that an assert fixes the bug? 

Users use relative paths that vite resolves, see https://github.com/inlang/inlang/blob/0c6c8b1080ff69ca638fef498eec512b824cc059/source-code/website/src/pages/index/sections/03-editor/index.tsx#L6-L9

<img width="685" alt="CleanShot 2023-04-20 at 13 33 45@2x" src="https://user-images.githubusercontent.com/35429197/233354124-0ab94aab-143b-4928-bff3-def277dd82a1.png">

**Comment by brillout** [comment]
It's the second line of the patch that tentatively fixes the problem. I'll release in a moment; we'll see whether it fixes it 😉.

**Comment by brillout** [comment]
Fix released in `0.4.115`.

**Comment by samuelstroschein** [comment]
Works
